wiki/Beatmap_Editor/Beat_Snap_Divisor/en.md

```diff
@@ -1,123 +1,113 @@
-# Beat snap divisor
+# Beat Snap Divisor
 
-The Beat Snap Divisor is used to limit where notes can be placed so that they always fall on the beat. Moving the slider to the right increases the granularity of the notes, that is, more notes can be placed in the same measure.
+![Beat Snap Divisor in the editor](img/beat_snap_divisor.png)
 
-Each of the settings put different colours of ticks on the [Timelines](/wiki/Beatmap_Editor/Timelines). The ticks are as follows (for [Standard](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch)):
+The **Beat Snap Divisor** is a built-in editor functionality that ensures all [hit objects](/wiki/Hit_Object) are placed proportionally to the song's beats. It can be found in the upper-right portion of the editor screen, where it corresponds to the granularity of the objects on the [editor timeline](/wiki/Beatmap_Editor/Timelines); that is, a denser Beat Snap Divisor setting allows for more notes to be placed in the same measure and vice versa. 
```

Comment:
It might be worth separating what the "divisor" is and what "beat snap" is. I wouldn't call "beat snap divisor" editor functionality; just "beat snap" is the better term for that. Also might be worth mentioning that it's basically a quantizing implementation (the music term for what we're doing here).

"Beat Snap" = editor functionality to ensure beats are snapped/quantised
"Beat Snap Divisor" = the division to user for the above system.